config/routes.rb

```diff
@@ -11,12 +11,11 @@
   if HostingEnvironment.sandbox_mode?
     root to: 'content#sandbox'
   else
-    root to: redirect('/candidate')
+    root to: redirect('/candidate/account')
   end
 
   namespace :candidate_interface, path: '/candidate' do
-    get '/' => 'start_page#show', as: :start
-
+    get '/', to: redirect('https://www.gov.uk/apply-for-teacher-training')
```

Comment:
The GovUK Apply start page url is littered throughout the code. 
Would it make sense to make this a global via an initialiser?